src/coreclr/tools/Common/TypeSystem/Interop/IL/PInvokeDelegateWrapper.cs

```diff
@@ -83,6 +83,14 @@ public override MetadataType MetadataBaseType
             }
         }
 
+        public override DefType BaseType
```

Comment:
This was a bug that I found because the methods on this type started to be stripped. Ooops.